org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corrections/proposals/LocalCorrectionsSubProcessor.java

```diff
@@ -1130,4 +1127,131 @@ public static void addValueForAnnotationProposals(IInvocationContext context, IP
 		}
 	}
 
+	/* (non-Javadoc)

```

Comment:
As a todo, I'll probably be implementing all of these in a later PR. It'll be a lot of manual verification.